docs/TELEMETRY.md

```diff
@@ -41,26 +41,10 @@ telemetry: {
   "data": {
     "attributes": {
       "etld":"f231d141395abf6f4c98dd55fe8c37e2752e82d72e1ffd3b64bdc6c978692fc6",
-      "num_blockable_trackers":"2",
```

Comment:
According the PHD it seems like we still want this number? "# of trackers loaded (integer; could be 0?)" under https://docs.google.com/document/d/1JeCS71Kf0ZJivxgNrrIvpViHCwU-QOhKvH92Av0b7Yc/edit#heading=h.d47hkl2j4nbu